test/functional/feature_notifications.py

```diff
@@ -18,7 +18,7 @@
 # Windows disallow control characters (0-31) and /\?%:|"<>
 FILE_CHAR_START = 32 if os.name == 'nt' else 1
 FILE_CHAR_END = 128
-FILE_CHAR_BLOCKLIST = '/\\?%*:|"<>' if os.name == 'nt' else '/'
+FILE_CHARS_DISALLOWED = '/\\?%*:|"<>' if os.name == 'nt' else '/'
```

Comment:
FILE_CHARS_BLACKLIST was ok, and it reverts what should not have been allowed on the first place. Please restore **as it was before**

Reply:
There's no point in repeating yourself. This change is good on its own if even it was changing from blacklist to disallowed.

Reply:
Now it will look like all this drama was only because people wanted a variable name to clearer in the first place.

Indeed "disallowed" is clearer, but imagine if the initial PR was changing "blacklist" to "disallowed"? It would have been a political attack anyway.

And then someone would create a PR reverting the change, and in the middle of the PR drama he would change it "blocklist" and everybody would be fine?

Is it common in this repository for pull requests to be opened changing names of a single variable?